src/aap_eda/settings/default.py

```diff
@@ -696,3 +697,10 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
 API_PATH_TO_UI_PATH_MAP = settings.get(
     "API_PATH_UI_PATH_MAP", {"/api/controller": "/execution", "/": "/#"}
 )
+
+
+ANSIBLE_LOCAL_TEMP = settings.get("ANSIBLE_LOCAL_TEMP", tempfile.gettempdir())
+if not os.access(ANSIBLE_LOCAL_TEMP, os.W_OK):
+    raise ImproperlyConfigured(
+        f"ANSIBLE_LOCAL_TEMP ${ANSIBLE_LOCAL_TEMP} is not writable"
+    )
```

Comment:
This can end up in a misleading message because `ANSIBLE_LOCAL_TEMP` could not be defined and the default value not writable. Also the message has a misleading `$` char, maybe a typo from bash syntax? 

I suggest instead:

```suggestion
    raise ImproperlyConfigured(
        f"{ANSIBLE_LOCAL_TEMP} is not writable, please define ANSIBLE_LOCAL_TEMP with a writable directory."
    )
```